app/views/components/docs/navigation-sidebar.yml

```diff
@@ -0,0 +1,130 @@
+name: Navigation Sidebar
+description: Sidebar showing related content, including topics, guidance, collections and policies (where applicable)
+accessibility_criteria: |
+  - Should have a role of 'complementary' as it complements the main page content
+  - Should have a role of 'navigation' on any navigation elements inside the component
+shared_accessibility_criteria:
+  - link
+examples:
+  default:
+    data:
+      related_items:
+        - text: Find an apprenticeship
+          href: /apply-apprenticeship
```

Comment:
Genuine question - should we be using 'href' or 'path' for link href attributes? I was sure href was more appropriate but remember being told to use path at some point, so I've been using it since. What's the consensus?

Reply:
I know there's been some debate on this, I thought the conclusion was to use 'href' as that's what we've used in the Back to Top link, Print link, and Download link components. Happy to change if we decide that 'path' is better though

cc @nickcolley